examples/cached_alias_config.py

```diff
@@ -33,6 +33,7 @@
     assert await cache.get("key") == "value"
     assert isinstance(cache, SimpleMemoryCache)
     assert isinstance(cache.serializer, DefaultSerializer)
+    await cache.close()
```

Comment:
yup, its just for the sake of "good" example. Its not ideal to close a singleton cache in the middle of a cache :)